DESCRIPTION

```diff
@@ -31,16 +31,30 @@ Depends:
     teal.transform (>= 0.5.0.9015)
 Imports:
     checkmate (>= 2.1.0),
+    colourpicker,
     dplyr (>= 1.0.5),
     DT (>= 0.13),
     forcats (>= 1.0.0),
+    generics,
+    ggExtra,
+    ggpmisc (>= 0.4.3),
+    ggpp,
+    ggrepel,
+    goftest,
     grid,
+    gridExtra,
+    htmlwidgets,
+    jsonlite,
+    lattice (>= 0.18-4),
+    MASS,
+    rtables (>= 0.6.8),
```

Comment:
With all these additions this won't pass CRAN checks of at most 20 non base dependencies! See this issue for tmc: https://github.com/insightsengineering/coredev-tasks/issues/604
I know this goes against the purpose of this PR or the issue #518 but I think we need to keep it in mind. 

Can some of them be removed? For example, gridExtra already imports grid, so can grid be omitted from here (not sure if it can be done or not)?

Reply:
That note from `CRAN` checks was the reason these packages were initially moved to `Suggests`. However, it's not a hard requirement.

With {tmc} we have ~27 non-base dependencies and some time ago we found packages in cran that have around the same number of deps or more.

If this is rejected we could try to argue the necessity, as these are real dependencies that are being used and have an effect on its usage (see library calls in teal.gallery)

We will be able to remove `lifecycle` when integrating the new changes from `tmc`, but others are very hard to replace.

The alternative is make them truly soft-dependencies

Reply:
> That note from `CRAN` checks was the reason these packages were initially moved to `Suggests`.
> However, it's not a hard requirement.
I thought it was that reason that made you move to Suggests. 

The requirement is not very strict for now...
I guess time will tell